doc/user-guide/Getting-started.md

```diff
@@ -55,6 +85,22 @@ After registration, the user will connect automatically.
 
 Registered users wishing to add an existing account to Adium should enter the MongooseIM server’s IP address in the **Connect Server** field on the **Options** tab.
 
+### Pidgin (Ubuntu & CentOS)
+
+1. 	Launch Pidgin.
+2. 	Click the Add button to configure your account.
+3. 	In the **Basic** tab choose **XMPP** as protocol,
+		then enter Username, Domain and Password and click **Add**.
```

Comment:
We usually stick to a "one sentence per line" convention in docs (without characters per line limit), so no need to break it into several lines.